src/common_c.py

```diff
@@ -495,8 +495,8 @@ def generate_json_common_c(out):
 		  (long long int) map->keys[i]);
       if (nret < 0 || (size_t) nret >= sizeof (numstr))
 	{
-	  if (!*err && asprintf (err, "Error to print string") < 0)
-	    *(err) = strdup ("error allocating memory");
+	  if (!*err)
+            *err = strdup("Error to print string");
```

Comment:
please add a space between strdup and '('

Reply:
all done